src/main/java/com/spotinst/sdkjava/model/api/azure/elastiGroup/V3/ApiElastigroupSuspendedProcessAzure.java

```diff
@@ -0,0 +1,62 @@
+package com.spotinst.sdkjava.model.api.azure.elastiGroup.V3;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+import com.spotinst.sdkjava.enums.ProcessNameEnumsAzure;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiElastigroupSuspendedProcessAzure implements IPartialUpdateEntity {
```

Comment:
Change naming convention to reflect which request or response it belongs to.

Reply:
changed the naming convention